### PR TITLE
added api specs and overview

### PR DESCRIPTION
**Pull request by kimerran**
for review

### CODE REVIEW

File: docs/api-spec/post-votes-by-comment.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+`commentID`
+
+# request
+```
+n/a
+```
+
+# response
+```
+{ "result": "ok" }
```
Comment (ghost) [inline docs/api-spec/post-votes-by-comment.md]: For consistency and verbosity, please don't use single-line examples. See https://github.com/platf0rm/platform/pull/9/commits/b997e368b5aeba61829257987a01defda3938f08#diff-03db1675cc9d339a9c38c4af5f3c5a05 as an example.

**Review by ghost — CHANGES_REQUESTED** [review]
Please put requirements for the parameters such as data types.

**Review by ac-opensource — APPROVED** [review]
LGTM after fixing @vkeusebio's comment

**Comment by kimerran** [comment]
@vkeusebio I made some [formatting changes](https://github.com/platf0rm/platform/pull/9/commits/3e88d957daf42eebd75ee181ec7855ca6dabd9c2). please review again. thanks

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
@kimerran LGTM, merging. Thanks!

**Pull request merged**